loadtest/control/simplecontroller/utils_test.go

```diff
@@ -0,0 +1,16 @@
+package simplecontroller
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestRandomizeUserName(t *testing.T) {
+
+	name := randomizeUserName("test-agent-1-user-4")
+	assert.Contains(t, name, "test-agent-1-user")
+
+	name = randomizeUserName("lt1-user4")
+	assert.Contains(t, name, "lt1-user")
+}
```

Comment:
A couple of suggestions:

- I'd prefer to be specific by making sure the resulting string actually starts with the expected substring.
- Maybe add a test for the case where the pattern is not found and the input string is returned unchanged.

Reply:
Sure, I can increase test cases 👍